src/DataCollector/RouteCollector.php

```diff
@@ -5,8 +5,6 @@
 use Closure;
 use DebugBar\DataCollector\DataCollector;
 use DebugBar\DataCollector\Renderable;
-use Illuminate\Http\Request;
-use Illuminate\Routing\Route;
```

Comment:
Yes, it can be removed, since the import is not used anywhere, but I would suggest to keep the import of the Response class and use it here
https://github.com/barryvdh/laravel-debugbar/blob/4a0e5842456819caafad7e5195b2bfe880a95d6d/src/DataCollector/RouteCollector.php#L72
as Response::class instead of a string value

Reply:
you're free to make a separate PR, but I'm going to leave this as is.